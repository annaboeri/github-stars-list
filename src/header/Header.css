/* Header Styles */
.header {
  align-items: center;
  background-color: var(--gray-dark);
  color: white;
  display: flex;
  justify-content: center;
  padding: 30px 0;
}

.logo-container {
  margin: 20px;
}

@media screen and (max-width: 40em) {
  .header {
    flex-direction: column;
    padding: 20px 10px;
  }
  .logo-conatiner {
    margin-bottom: 0;
    width: 75px;
  }
}
