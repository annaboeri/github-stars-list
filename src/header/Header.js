import React from 'react';
import logo from '../github-logo.png';
import './Header.css';

const Header = () => {
  return (
    <header className="header">
      <a href="https://github.com/" className="logo-container">
        <img src={logo} alt="github logo" />
      </a>
      <div>
        <h1>Github Stars</h1>
        <h2>List of the Most Popular Github Repos</h2>
      </div>
    </header>
  );
}

export default Header;
