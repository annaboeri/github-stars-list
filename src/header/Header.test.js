import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './Header';

describe('Header Component', () => {
  it('should render correctly', (done) => {

    const component = shallow(<Header />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
    expect(component.find('.header').exists()).toBe(true);
    done();
  });
});
