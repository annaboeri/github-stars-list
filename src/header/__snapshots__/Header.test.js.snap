// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<header
  className="header"
>
  <a
    className="logo-container"
    href="https://github.com/"
  >
    <img
      alt="github logo"
      src="github-logo.png"
    />
  </a>
  <div>
    <h1>
      Github Stars
    </h1>
    <h2>
      List of the Most Popular Github Repos
    </h2>
  </div>
</header>
`;
